mne/viz/tests/test_3d.py

```diff
@@ -681,10 +682,17 @@ def test_plot_volume_source_estimates_morph():
                  clim=dict(lims=[-1, 2, 3], kind='value'))
 
 
+bad_azure_3d = pytest.mark.skipif(
+    os.getenv('AZURE_CI_WINDOWS', 'false') == 'true' and
+    sys.version_info[:2] == (3, 8),
+    reason='Crashes workers on Azure')
```

Comment:
FYI @GuillaumeFavelier needed to add a couple of skips here because of this

https://dev.azure.com/mne-tools/mne-python/_build/results?buildId=7171&view=logs&j=c0130b29-789d-5a3c-6978-10796a508a7f&t=5f931690-334a-58bc-2c61-e2793b49abfb